tools/sailfish/sailfish.xml

```diff
@@ -56,10 +67,24 @@
             --index $index_path
             #if $single_or_paired.single_or_paired_opts == 'single':
                 --libType ${single_or_paired.strandedness}
-                --unmatedReads ./single.$ext
+                #if $compressed == 'GZ':
+                    --unmatedReads <(zcat ./single.$ext)
+                #else if $compressed == 'BZ2':
+                    --unmatedReads <(bzcat ./single.$ext)
```

Comment:
Is there a Galaxy bzip2 package even? If not then these become conda only tools.